### PR TITLE
Coding Challenge

### PR DESCRIPTION
**Pull request by rin0o1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-## IBM MQ samples and patterns
+## IBM MQ samples and patterns 
+Please Note: the instructions below should NOT be considered for ibm-messagin-mq-cloud-showcase-app
```
Comment (chughts) [inline README.md:2]: The repo readme needs an overall. Remove this change from the pull request, as it is misleading and reads "you will spend countless hours learning the basics with the showcase app"

File: ibm-messaging-mq-cloud-showcase-app/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/LICENSE:1]: Please remove this license as it duplicates the main repo license.

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/package.json [hunk 3]
```diff
@@ -26,16 +26,16 @@
     "react": "16.10.0",
     "react-dom": "16.10.0",
     "react-flow-renderer": "^10.3.17",
-    "react-router-dom": "5.3.4",
+    "react-router-dom": "5.0.0",
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/package.json:29]: Please pull this file from the repo and test with the package as updated for security fixes. 

As this pull request regresses those fixes.

Comment (chughts) [reply ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/package.json:29]: This also applies to the package-lock.json file.

**Review by chughts — CHANGES_REQUESTED** [review]

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/package.json (hunk 3 above)
Comment (rin0o1) [reply ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/package.json:29]: Just tested and it works with both the react-router-dom "5.3.4" & "5.0.0"

**Comment by rin0o1** [comment]
Changes applied

**Review by chughts — APPROVED** [review]

**Pull request merged**